### PR TITLE
Reviews now sorted by recency and upvotes

### PR DESCRIPTION
**Pull request by MichaelDimitras**
I refactored my database code to query top reviews and recent reviews separately.  Before it was returning all the reviews for a particular product.

### CODE REVIEW

File: database/db.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const { Pool } = require('pg');
 
 const config = {
   user: 'michaeldimitras', // env var: PGUSER
-  database: 'michaeldimitras', // env var: PGDATABASE
+  database: 'reviews', // env var: PGDATABASE
```
Comment (TomGeshury) [inline database/db.js:5]: Should be called shamazondb

File: client/src/index.jsx [hunk 2]
```diff
@@ -41,28 +41,29 @@ class App extends React.Component {
...
 
   getRecentReviews() {
     //Return the most recent reviews
     //To be used in the '.recent-reviews' section on the right of the page
-    return this.state.reviews.slice(0,10);
+    return this.state.reviews.recent;
   }
 
   render() {
+    console.log(this.state.reviews)
```
Comment (TomGeshury) [inline client/src/index.jsx]: needless console.log

File: client/src/index.jsx [hunk 3]
```diff
@@ -30,9 +30,9 @@ class App extends React.Component {
       dataType: 'json',
       success: (data, status) => {
         if(status === 'success') {
-          this.setState({reviews: data})
+          this.setState({reviews: data});
         } else {
-          this.setState({reviews: []})
+          this.setState({recent: [], top: [], count: []});
```
Comment (TomGeshury) [inline client/src/index.jsx]: top and recent is fairly confusing. Change to recentReviews and topReviews (?)

File: database/db.js (hunk 1 above)
Comment (MichaelDimitras) [reply database/db.js:5]: I will fix for the proxy

**Review by TomGeshury — APPROVED** [review]

**Pull request merged**